Make api prefix configurable in mimirtool (#2918)
grafana/mimir: docs/sources/operators-guide/tools/mimirtool.md

The following commands interact with Grafana Mimir Alertmanager configuration and alert template files.

#### Configuration

This configuration applies to all `mimirtool alerts` and `mimirtool alertmanager` commands.

| Environment variable | Flag           | Description                                                                |
| -------------------- | -------------- | -------------------------------------------------------------------------- |
| `MIMIR_API_PREFIX`   | `--api-prefix` | Sets the prefix for the Grafana Mimir endpoints. Default is '/prometheus'. |

> 

Reviewer: I left a comment to remove this from the `alertmanager` command. You'll also need to update the docs - remove the mention of alertmanager and move this to the `alerts verify` command

mimirtool alerts verify
```

#### Configuration

> This configuration applies to all `mimirtool alerts` and `mimirtool alertmanager` commands.

Reviewer: This is no longer true
```suggestion
```

- Interact with individual rule groups in the Mimir ruler
- Manipulate local rule files

> ##### Configuration

Reviewer: ```suggestion
#### Configuration
```

Reviewer: Can you add a sentence mentioning that this configuration applies to all the `mimirtool rules` commands?

| Environment variable | Flag           | Description                                                                                                                                        |
| -------------------- | -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `MIMIR_API_PREFIX`   | `--api-prefix` | sets the prefix for the prometheus endpoint. Default is '/prometheus'                                                                              |

Reviewer: ```suggestion
| `MIMIR_API_PREFIX`   | `--api-prefix` | Sets the prefix for the Grafana Mimir endpoints. Default is '/prometheus'.                                                                              |
```
